### PR TITLE
Hooks: Qt WebEngine

### PR DESCRIPTION
**Pull request by xoviat**
Since no one took this up, let's see how much work this is going to be...

### CODE REVIEW

File: PyInstaller/hooks/hook-PyQt5.py [hunk 1]
```diff
@@ -6,26 +6,19 @@
...
+from PyInstaller.utils.hooks import (get_module_attribute,
+    is_module_satisfies, qt_menu_nib_dir, collect_data_files)
+from PyInstaller.compat import is_darwin
 
-# On Windows system PATH has to be extended to point to the PyQt5 directory.
-# The PySide directory contains Qt dlls. We need to avoid including different
-# version of Qt libraries when there is installed another application (e.g. QtCreator)
-if is_win:
-    from PyInstaller.utils.win32.winutils import extend_system_path
-    extend_system_path([os.path.join(x, 'PyQt5') for x in getsitepackages()])
```
Comment (xoviat) [inline PyInstaller/hooks/hook-PyQt5.py]: Not sure why this was removed, may need to revert...

File: PyInstaller/loader/rthooks.dat [hunk 2]
```diff
@@ -14,7 +14,7 @@
     'osgeo':      ['pyi_rth_osgeo.py'],
     'pkg_resources':  ['pyi_rth_pkgres.py'],
     'PyQt4':      ['pyi_rth_qt4plugins.py'],
-    'PyQt5':      ['pyi_rth_qt5.py', 'pyi_rth_qt5plugins.py'],
+    'PyQt5':      ['pyi_rth_qt5.py'],
```
Comment (xoviat) [inline PyInstaller/loader/rthooks.dat]: Also, may need to revert changes here.

**Comment by xoviat** [comment]
Note: this branch may conflict with #2403, which should be merged into this pull request.

**Comment by xoviat** [comment]
Reopened to cancel CI.

**Comment by xoviat** [comment]
@htgoebel Should be ready to merge. Fixes some, but not all cases.

**Comment by htgoebel** [comment]
I was about to merge this, but this pull-request contains a lot of unrelated changes. Would you mind updating it to only include the relevant ones? Thanks.

**Comment by xoviat** [comment]
The unrelated changes are from the ox branch. One you merge that PR, the changes should disappear after a rebase.

**Comment by htgoebel** [comment]
Now the osx-branch was merged, could you please clean up this one. I'm a bit lost in what is related to this pull-request and what belongs to others. Thanks.

**Comment by xoviat** [comment]
@htgoebel Done.

**Pull request merged**